ot-node.js

```diff
@@ -154,8 +154,9 @@ class OTNode {
         try {
             await graphStorage.connect();
             log.info(`Connected to graph database: ${graphStorage.identify()}`);
-            const myVersion = await graphStorage.version();
-            log.info(`Database version: ${myVersion}`);
+            // TODO https://www.pivotaltracker.com/story/show/157873617
```

Comment:
git rebase is messing up with me, but this is already on develop so should not make any change!